### DISCUSSION TITLE
Nexstim TMS coil ccd file

### DISCUSSION THREAD

**Discussion by prem-ganesh23** [simnibs/simnibs#378] [8. Simulation workflows]
Currently working on a modeling project using the Nexstim NBR1.0 system with the Nexstim Cooled Coil. Is there a coil definition file available or are there any guidelines for making one?

Thanks!

**Comment by diedrim** [comment]
You would need to ask the company about the geometry details about the coil. Once you have this information, we could help you further.

**Reply by prem-ganesh23** [reply]
At this point I have theoretical maximum dI/dt at the start of the pulse, dI/dt over the first 1/4 of the pulse, mean and maximum winding diameter, and biphasic pulse length. Knowing this info (and rest still pending an agreement with the company) I'd like to make a definition file by assuming the other geometry is the same as a Magstim D70 but scaled to our coil size and using the information we do know. Is there a recommended way to go this route?
